src/main/java/com/google/api/gax/grpc/InstantiatingChannelProvider.java

```diff
@@ -185,7 +193,7 @@ public static Builder newBuilder() {
 
     // Default names and versions of the client and the service generator.
     private static final String DEFAULT_GENERATOR_NAME = "gapic";
-    private static final String DEFAULT_CLIENT_LIB_NAME = "gax";
+    private static final String DEFAULT_CLIENT_LIB_NAME = "default";
```

Comment:
Changed to gapic. According to the doc, lib2 is used by Veneer layer. For Vkit layer we use gapic.